### DISCUSSION TITLE
Is wake on wifi available?

### DISCUSSION THREAD

**Discussion by PrzemekSkw** [seriousm4x/UpSnap#68] [Help wanted]
Hi, I enable wake on wifi (WoWLAN) on my laptop and it looks OK but it don't work in Upsnap.
```
iw phy0 wowlan show
WoWLAN is enabled:
 * wake up on disconnect
 * wake up on magic packet
```
I enable on my laptop with that tutorial: [link](https://www.cyberciti.biz/faq/configure-wireless-wake-on-lan-for-linux-wifi-wowlan-card/)

I can make laptop sleep with that command: `sudo sh -c 'echo mem > /sys/power/state'` but don't know how to implement it in upsnap.

Nothing happen when I push button in UpSnap dashboard.
Regards.

**Comment by seriousm4x** [comment]
Hi, thats something I've not tested as I don't have any hardware supporting wake on wifi. From what I've read about WoWLAN after a quick google search is, that the implementation should be the same as normal wake on lan (but there is very few documentation online). Have you tried it with another program if that can wake your laptop?

For sending your laptop to sleep you can use the following as the `Shutdown cmd`:
```bash
sshpass -p your_password ssh -o "StrictHostKeyChecking=no" user@hostname 'sudo sh -c "echo mem > /sys/power/state'"'
```

**Comment by PrzemekSkw** [comment]
I try with that command but also don't work. But it's OK I will don't use that entry. More important is how to set Wake Cron. Can You help me? I want to set to wake may server on 7:00. How to setup that in UpSnap? Just put this in field: `0 7 * * *`?

**Reply by seriousm4x** [reply]
Yes, currect. [`https://crontab.guru/#0_7_*_*_*`](https://crontab.guru/#0_7_*_*_*)
Make sure to toggle the `Enable` switch next to the input field.

**Comment by PrzemekSkw** [comment]
Yes, I set them. I setup like that:
![image](https://user-images.githubusercontent.com/25219988/231192044-190ffea4-6449-475c-9e8a-5dab8f9c52ad.png)
![image](https://user-images.githubusercontent.com/25219988/231192125-db663f05-bba2-46c6-a473-ab0ced8c30a5.png)
 to sleep on 01:00 and wake up on 7:00 every day.
 Thanks for help.

**Comment by PrzemekSkw** [comment]
@seriousm4x Hi, unfortunately today at 7 it don't wake up my server I have to manually click that button in upsnap panel. Do You know what may cause that?
